### ISSUE TITLE
Reduce package size

### ISSUE THREAD

**Issue by repeatedly** [clear-code/td-agent-builder#44]
We can remove unnecessary files from package to reduce the installed size.
One example is `/opt/td-agent/lib/ruby/gems/2.4.0/cache`.

In addition, I noticed rpm package is 2x bigger than deb.
The size of binary is larger. For example, jemalloc and ruby are different.

- Ubuntu 18.04

```
$ ls -ahl /opt/td-agent/lib/
total 5.5M
drwxr-xr-x 4 root root 4.0K Feb 18 02:45 .
drwxr-xr-x 7 root root 4.0K Feb 18 02:45 ..
-rw-r--r-- 1 root root 912K May  7 14:45 libjemalloc.a
-rw-r--r-- 1 root root 912K May  7 14:45 libjemalloc_pic.a
lrwxrwxrwx 1 root root   16 May  7 14:45 libjemalloc.so -> libjemalloc.so.2
-rw-r--r-- 1 root root 552K May  7 14:45 libjemalloc.so.2
lrwxrwxrwx 1 root root   16 May  7 14:45 libruby.so -> libruby.so.2.7.1
lrwxrwxrwx 1 root root   16 May  7 14:45 libruby.so.2.7 -> libruby.so.2.7.1
-rw-r--r-- 1 root root 3.2M May  7 14:45 libruby.so.2.7.1
drwxr-xr-x 2 root root 4.0K Feb 18 02:45 pkgconfig
drwxr-xr-x 6 root root 4.0K Feb 18 02:45 ruby
```

- CentOS 8

```
$ ls -ahl /opt/td-agent/lib/
total 65M
drwxr-xr-x. 4 root root  191 Apr 13 13:15 .
drwxr-xr-x. 7 root root   72 Apr 13 13:15 ..
-rwxr-xr-x. 1 root root  25M May  8  2020 libjemalloc.a
lrwxrwxrwx. 1 root root   16 May  8  2020 libjemalloc.so -> libjemalloc.so.2
-rwxr-xr-x. 1 root root 5.8M May  8  2020 libjemalloc.so.2
-rwxr-xr-x. 1 root root  25M May  8  2020 libjemalloc_pic.a
lrwxrwxrwx. 1 root root   16 May  8  2020 libruby.so -> libruby.so.2.7.1
lrwxrwxrwx. 1 root root   16 May  8  2020 libruby.so.2.7 -> libruby.so.2.7.1
-rwxr-xr-x. 1 root root 9.6M May  8  2020 libruby.so.2.7.1
drwxr-xr-x. 2 root root   44 Apr 13 13:15 pkgconfig
drwxr-xr-x. 6 root root   67 Apr 13 13:15 ruby
```

Of course, other extension gems are also bigger.

**Comment by ashie** [comment]
> In addition, I noticed rpm package is 2x bigger than deb.

I've found that debuginfo packages aren't built by td-agent-builder.
So that strip command isn't performed against executables.
I've fixed this issue in #50.

> We can remove unnecessary files from package to reduce the installed size.
> One example is /opt/td-agent/lib/ruby/gems/2.4.0/cache.

Removed files in omnibus-td-agent: https://github.com/treasure-data/omnibus-td-agent/blob/05a1f1a43732383828f4cbec47e393773c14acea/config/software/td-agent-cleanup.rb

**Comment by repeatedly** [comment]
I confirmed rpm size is now small :)
